examples/deadline/All-In-AWS-Infrastructure-SEP/ts/README.md

```diff
@@ -0,0 +1,49 @@
+# RFDK Sample Application - Deadline - Typescript
+
+## Overview
+[Back to overview](../README.md)
+
+## Instructions
+
+---
+**NOTE**
+
+These instructions assume that your working directory is `examples/deadline/All-In-AWS-Infrastructure-SEP/ts/` relative to the root of the RFDK package.
+
+---
+1. This sample app on the mainline branch may contain features that have not yet been officially released, and may not be available in the aws-rfdk package installed through npm from npmjs. To work from an example of the latest release, please switch to the release branch. If you would like to try out unreleased features, you can stay on mainline and follow the instructions for building, packing, and installing the aws-rfdk from your local repository.
```

Comment:
This still mentions needing to pack and install the local version, which isn't required.